<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>281</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_userName">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用 户 名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_userName">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_password">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>11</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>密    码</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_password">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_confrimPassword">
        <property name="font">
         <font>
          <family>黑体</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确认密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_confrimPassword">
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="userNameRepeat">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
    </font>
   </property>
   <property name="text">
    <string>用户名错误</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordDifferent">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>密码不一致</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tipExist">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户已经存在</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_complete">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>194</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>完成注册</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="pushButton_back">
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_adminComplete">
      <property name="font">
       <font>
        <family>黑体</family>
       </font>
      </property>
      <property name="text">
       <string>管理员注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
